pkg/apis/agones/v1/gameserver_test.go

```diff
@@ -205,17 +212,19 @@ func TestGameServerApplyDefaults(t *testing.T) {
 							Protocol:      corev1.ProtocolTCP,
 						},
 					},
-					Health: Health{Disabled: true},
+					Health:  Health{Disabled: true},
+					Logging: Logging{SdkServer: "Debug"},
```

Comment:
rather than piggyback on an existing test case ("convert from legacy single port to multiple" to test applying a custom log level, please add a new test case that specifically tests setting the log level.

Reply:
OK, adding a separate test